index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Mobile First</title>
+  </head>
+
+  <body>
+    <!-- Header-section starts here -->
+ <div class="hero-section">
+
+    <header>
+      <div class="container">
+       
+        <div class="mobile-status">
+          <div class="left-side">
+            <h4>9:14</h4>
+          </div>
+          <div class="right-side">
+              <img src="/images/status-icons.svg" alt="status icons" />
+            </div>
+          </div>
+          <nav class="navbar">
+            <div class="logo">
+              <h3>MOZO</h3>
+            </div>
+            <div class="menu-bar">
+              <img src="/images/menu-bar.png" alt="menu-bar" />
+            </div>
+          </nav>
+        </div>
+      </header>
+
+       <!-- Header-section Ends here -->
+      <!-- Home-page section Starts here -->
+     <main>
+        <section class="home-page container">
+          <div class="home-title">
+            <h1>
+              I’m Felix <br />
+              Glad to see you!
+            </h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+             <a href="#" class="connect">LET'S CONNECT</a>
+
+             <ul class="social-icons">
+
+              <li>
+                <a href="twitter.com/" target="_blank">
+               <img src="/images/twitter.svg" alt="twitter-icon">
+                </a>
+              </li>
+
+              <li>
+                <a href="linkedin.com" target="_blank">
+              <img src="/images/linkedin.svg" alt="linkedin-icon"></a>
+              </li>
+
+              <li>
+                <a href="medium.com" target="_blank">
+               <img src="/images/medium.svg" alt="medium-icon">
+                </a>
+              </li>
+
+              <li>
+                <a href="github.com" target="_blank">
+                <img src="/images/github.svg" alt="github-icon">
+                </a>
+              </li>
+
+              <li>
+                <a href="angel.com" target="_blank">
+                  <img src="/images/angel.svg" alt="angel-icon">
+                </a>
+              </li>
+
```

Comment:
- The same issue solution is applicable here for the images. let's update the path by adding a `.` before the `/images` to fix this issue. kindly do the same for all other images.
`  <img src="./images/angel.svg" alt="angel-icon">`